docs/eventing/samples/sequence/sequence-reply-to-event-display/README.md

```diff
@@ -0,0 +1,194 @@
+---
+title: "Sequence Wired to event-display"
+weight: 20
+type: "docs"
+---
+
+# Using Sequences in series
+
+## Prerequisites
+
+For this example, we'll assume you have set up an `InMemoryChannel`
+as well as Knative Serving (for our functions). The examples use `default`
+namespace, again, if you want to deploy to another Namespace, you will need
+to modify the examples to reflect this.
+
+If you want to use different type of `Channel`, you will have to modify the
+`Sequence.Spec.ChannelTemplate` to create the appropriate Channel resources.
+
+## Overview
+
+We are going to create the following logical configuration. We create a CronJobSource,
+feeding events to a `Sequence`, then taking the output of that `Sequence` and
+displaying the resulting output.
+
+![Logical Configuration](./sequence-reply-to-event-display.png)
```

Comment:
The links to this image aren't resolving in either file.

Reply:
Is this because of the diff view, or something else? In our other markdown files, this is how we seem to be embedding images inline, how should I be doing them if not this? I just looked at this for example:
/docs/eventing/README.md
has this:
![Control plane object model](./images/control-plane.png)

I'll play with it some more.

Reply:
Hmm, maybe it is just because of the diff view. I thought images with correct links would resolve appropriately, but this link looks correct and it's still not loading.